kbase-extension/static/kbase/js/common/lang.js

```diff
@@ -7,18 +7,80 @@ define(['bluebird'], (Promise) => {
         }
     }
 
-    function pRequire(require, module) {
+    /**
+     * This wraps a require call in a bluebird Promise. The Promise then resolves
+     * into an array of Widgets, one for each.
+     * @param {Array} modules an array of module paths for a require call.
+     * @returns a Promise that resolves into an array of modules.
+     */
+    function pRequire(modules) {
         return new Promise((resolve, reject) => {
-            require(module, function () {
-                resolve(arguments);
+            require(modules, (...args) => {
+                resolve(args);
             }, (err) => {
                 reject(err);
             });
         });
     }
 
+    function isEqual(v1, v2) {
```

Comment:
copied, and slightly adjusted to remove unreachable if-branches, from `unodep`

Reply:
we have underscore as one of the deps -- maybe we should just use the underscore .isEqual function?

Reply:
Yeah, that sounds reasonable. I didn't think of that.
We should update to the latest underscore, too.